packages/loader/container-utils/src/error.ts

```diff
@@ -165,18 +167,17 @@ export class DataProcessingError extends LoggingError implements IErrorBase, IFl
 
         const normalizedError = normalizeError(originalError, { props });
 
-        if (!isExternalError(normalizedError)) {
-            return normalizedError;
-        }
-
-        // Create a new DataProcessingError to wrap this external error
-        const dataProcessingError =
-            wrapError(normalizedError, (message: string) => new DataProcessingError(message));
+        if (isExternalError(normalizedError) || isNormalizedLoggingError(normalizedError)) {
```

Comment:
Ok - What this boils down to is any External error or any error with `errorType === "genericError"` should get wrapped.  I think rather than exporting `isNormalizedLoggingError` which checks for that errorType, this code should own this logic.  This is more "honest" - this code says "I don't care why your errorType is "genericError", I will always wrap you", and we can get rid of `isNormalizedLoggingError` since it's not really true - see my comment down there, about the `GenericError` class.

Future note:  Once `normalizeError` stops putting errorType on, this will be simpler - if it's an externalError or has _no_ errorType, then wrap it.  (although for back-compat we'll probably still squash "genericError" errorType).

Reply:
Sounds good. I have added the changes.